libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -836,34 +907,46 @@ bool AC_WPNav::set_spline_destination_next_NEU_cm(const Vector3f& next_destinati
 // convert location to vector from ekf origin.  is_terrain_alt is set to true if resulting vector's z-axis should be treated as alt-above-terrain
 //      returns false if conversion failed (likely because terrain data was not available)
 bool AC_WPNav::get_vector_NEU_cm(const Location &loc, Vector3f &pos_from_origin_neu_cm, bool &is_terrain_alt)
+{
+    Vector3f pos_from_origin_neu_m = pos_from_origin_neu_cm * 0.01;
```

Comment:
I don't think we need to initialise pos_from_origin_neu_m.  We can assume that get_vector_NEU_m() will not change any of the arguments if it returns false.